### ISSUE TITLE
More elaborate examples. 

### ISSUE THREAD

**Issue by agarwalarpit** [tejasmanohar/node-schedule#160]
We need more elaborate examples for a quick and nice on-boarding experience for node developers.

**Comment by objectiveSee** [comment]
I agree with this, even for someone with experience it is very helpful to know how this module will be used. It's even more important than how it works :D

**Comment by tejasmanohar** [comment]
@agarwalarpit Did you get a chance to check out the wiki? https://github.com/tejasmanohar/node-schedule/wiki

@objectiveSee I agree. I'll add some basic documentation of how it works under the hood in the Wiki, but I think our source code is also pretty to the point- it'll get easier when our babel-node es6 is stable w/ v1 :+1:  Anything specific you're curious about?

**Comment by pgulley** [comment]
I agree- There are a lot of things in the code that are undocumented in the wiki.
I would have liked to see an example of creating Jobs without scheduling them, use of the 'scheduledJobs' object, Job names, to name a few things.

**Comment by mkozjak** [comment]
+1. And make a wiki single-page. This one is a hassle.

**Comment by tejasmanohar** [comment]
@mkozjak So a README?

**Comment by ljcp** [comment]
+1

**Comment by tejasmanohar** [comment]
the wiki is now community editable + you can send PRs to this project for README updates/changes if you think it's better having the main info in the READMe instead of the wiki. i will see how it's formatted and review the PR :smile:

**Comment by git0rhero** [comment]
Cron style Scheduling page first says the syntax has 6 items (* * * * * *)
Below examples all show 5 items only (* * * * *)
A bit of inconsistency here...
Which is correct? Both?

**Comment by s0l1dsn4k3** [comment]
Can someone write an example of a recurring cron task that executes once a month on the first day of the month, at some obvious time that I can make sense of? I have been trying to figure this out on my own, but the docs don't appear to support a monthly recurring cron task.

**Comment by tejasmanohar** [comment]
PRs wanted :)

**Comment by deltreey** [comment]
what happened to the wiki?

**Comment by tejasmanohar** [comment]
@deltreey see README

**Comment by sabali33** [comment]
> Can someone write an example of a recurring cron task that executes once a month on the first day of the month, at some obvious time that I can make sense of? I have been trying to figure this out on my own, but the docs don't appear to support a monthly recurring cron task.

I am experiencing the same issue. I can't make a monthly schedule at a date other than 1st day of a month
